extensions/amp-mega-menu/amp-mega-menu.md

```diff
@@ -0,0 +1,385 @@
+---
+$category@: layout
+formats:
+  - websites
+teaser:
+  text: Displays top-level navigational content inside expandable containers.
+experimental: true
+---
+
+<!--
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# `amp-mega-menu`
+
+A horizontal navigation bar containing a set of menu items that, when activated via click, open/close corresponding containers underneath with additional content.
```

Comment:
Simplify to: 

Horizontal navigation bar with menu items that open/close content containers on click.